CMakeLists.txt

```diff
@@ -87,7 +87,7 @@ else(MSVC)
   check_cxx_compiler_flag("-std=c++11"   SUPPORT_CXX11)
   check_cxx_compiler_flag("-std=c++0x"   SUPPORT_CXX0X)
   check_cxx_compiler_flag("-msse2"       SUPPORT_MSSE2)
-  set(CMAKE_C_FLAGS "-Wall -Wno-unknown-pragmas -fPIC -Wno-sign-compare")
+  set(CMAKE_C_FLAGS "-Wall -Werror -Wno-unknown-pragmas -fPIC -Wno-sign-compare")
```

Comment:
Is this intended to be part of the PR?

Reply:
This is something you'd need to discuss on dev. Currently between MSVC, Clang and gcc, some things generate errors and some don't.

Reply:
Ok, thanks for the feedback